app/src/main/java/com/android/unio/ui/authentication/Login.kt

```diff
@@ -1,14 +1,53 @@
 package com.android.unio.ui.authentication
 
-import android.annotation.SuppressLint
-import androidx.compose.material.Scaffold
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.android.unio.ui.navigation.NavigationAction
+import com.android.unio.ui.navigation.Screen
 
-@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
 @Composable
-fun LoginScreen() {
-  Scaffold(content = { Text("Login screen") }, modifier = Modifier.testTag("LoginScreen"))
+fun LoginScreen(navigationAction: NavigationAction) {
+  var email by remember { mutableStateOf("") }
+  var password by remember { mutableStateOf("") }
+  Scaffold(
+      modifier = Modifier.testTag("LoginScreen").fillMaxSize(),
+      content = { padding ->
+        Column(
+            modifier = Modifier.padding(padding).padding(vertical = 200.dp, horizontal = 50.dp),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally) {
+              OutlinedTextField(
+                  modifier = Modifier.padding(16.dp),
+                  label = { Text("Enter your university address") },
```

Comment:
It's a good habit to develop to avoid writing hard coded string in the code, and favor the use of String.xml resources. This will makes it easier to later [localize](https://developer.android.com/guide/topics/resources/localization) the application. It also makes the app more maintainable as it is easy to modify one text, without having to remember to modify every places that are displaying the hard coded text.